api/src/controllers/files.ts

```diff
@@ -140,16 +171,28 @@ router.post(
 			if (Array.isArray(keys) && keys.length > 1) {
 				const records = await service.readMany(keys, req.sanitizedQuery);
 
-				res.locals.payload = {
+				const payload = {
 					data: records,
 				};
+
+				if (res.locals.errors.length > 0) {
+					payload.errors = res.locals.errors;
```

Comment:
This doesn't expose the error message in the response, anyone have thoughts on a better way to do this?